examples/cli/cli_demo.go

```diff
@@ -59,11 +59,9 @@ func connect() {
 	config.SubscribeKey = "sub-c-26a73b0a-c3f2-11e9-8b24-569e8a5c3af3" //"demo" //"sub-c-10b61350-bec7-11e9-a375-f698c1d99dce" //"demo" //
 	//config.SecretKey = //"pam"    //"demo"
 
-	//config.PublishKey = "pub-c-cdea0ef1-c571-4b72-b43f-ff1dc8aa4c5d"
-	//config.SubscribeKey = "sub-c-4757f09c-c3f2-11e9-9d00-8a58a5558306"
-	//config.SecretKey = "sec-c-YTYxNzVjYzctNDY2MS00N2NmLTg2NjYtNGRlNWY1NjMxMDBm"
-
-	//config.AuthKey = "akey"
+	config.PublishKey = "pub-c-cdea0ef1-c571-4b72-b43f-ff1dc8aa4c5d"
```

Comment:
Should these be removed from the `examples`?

Reply:
these keys will be disabled, once the release is done. These keys are used in tests as well.

Reply:
Removed